package.json

```diff
@@ -130,13 +116,10 @@
       "optional": true
     }
   },
+  "packageManager": "pnpm@9.12.2",
   "engines": {
     "node": "18.* || 20.* || >= 21"
   },
-  "volta": {
```

Comment:
@NullVoxPopuli maybe e could keep volta config but pin pnpm using it?

Reply:
sure thing